src/libmongoc/doc/visual-studio-guide.rst

```diff
@@ -21,7 +23,9 @@ Right-click on your console application in the Solution Explorer and select "Pro
 .. image::
   static/msvc-add-include-directories.png
 
-(If you chose a different ``CMAKE_INSTALL_PREFIX`` :ref:`when you ran CMake <build-on-windows>`, your include paths will be different.)
+(If you chose a different ``$PREFIX``
+:doc:`when you installed mongo-c-driver </howto/source-install>`, your include
```

Comment:
Can/should we link to anchors (`#choosing-a-prefix`) if we can?

Reply:
In this case, probably yes.